autogtp/main.cpp

```diff
@@ -196,20 +198,28 @@ int main(int argc, char *argv[])
 #else
     QTextStream cerr(stderr, QIODevice::WriteOnly);
 #endif
-
     cerr << "autogtp v" << AUTOGTP_VERSION << endl;
 
+    auto start = std::chrono::high_resolution_clock::now();
     auto success = true;
-    auto games_played = 0;
+    long games_played = 0;
 
     do {
+        auto game_start = std::chrono::high_resolution_clock::now();
         QString netname;
-        success &= fetch_best_network_hash(cerr, netname);
-        success &= fetch_best_network(cerr, netname);
+        success &= fetch_best_network_hash(cerr, netname, games_played);
+        success &= fetch_best_network(cerr, netname, games_played);
         success &= run_one_game(cerr, netname);
         success &= upload_data(cerr, netname);
         games_played++;
-        cerr << games_played << " games played." << endl;
+
+        auto game_end = std::chrono::high_resolution_clock::now();
+        auto game_time_s = std::chrono::duration_cast<std::chrono::seconds>(game_end - game_start);
+        auto total_time_s = std::chrono::duration_cast<std::chrono::seconds>(game_end - start);
+        cerr << games_played << " games_played in "
+             << (total_time_s.count() / 60) << " minutes = "
```

Comment:
You can probably avoid the magic number here with std::chrono::minutes (but it's ok like this too).

Reply:
I use total minutes here, and seconds below so I need to convert between them (or use a bulky cast which I'm not a fan of)

Reply:
done.